mpris-widget/init.lua

```diff
@@ -68,7 +67,7 @@ function playerctl:watch(timeout, callback, widget)
             metadata.year = string.sub(words[8], 0, 4)
         end
 
-        callback(widget, metadata)
+        callback(cb_widget, metadata)
     end
 
     _, self.timer = awful.widget.watch(cmd, timeout, cb, widget)
```

Comment:
Not sure how to fix this one, I guess I need help 😄 

> mpris-widget/init.lua:73:5: setting non-standard global variable _

Reply:
I guess declaring a `local _` before this line, or assigning `self.timer = awful...()[2]` can do the trick.

However, it seems the code here wants to discard the widget and only use the timer from `awful.widget.watch`. Maybe the line should be replaced by `gears.timer`?

Reply:
```suggestion
    local _, timer = awful.widget.watch(cmd, timeout, cb, widget)
    self.timer = timer
```

It ain't pretty but gets the job done.

Reply:
Thank you!